### PR TITLE
Update the text as Gutenberg was officially released.

### PR DESCRIPTION
**Pull request by maximejobin**
## Description
Update the FAQ

## How has this been tested?
This is not code.

## Types of changes
Make the FAQ questions use the present instead of future verbe tense.

### CODE REVIEW

File: docs/designers-developers/faq.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Gutenberg is being developed on [GitHub](https://github.com/WordPress/gutenberg)
 
 ## When will Gutenberg be merged into WordPress?
 
-Gutenberg will be merged into WordPress 5.0, and will be released at the end of 2018, or early 2019. Please follow [WordPress.org News](https://wordpress.org/news/) for the latest information.
+Gutenberg was be merged into [WordPress 5.0](https://wordpress.org/news/2018/12/bebo/) that was released in December 2018.
```
Comment (chrisvanpatten) [inline docs/designers-developers/faq.md]:
```suggestion
Gutenberg was merged into [WordPress 5.0](https://wordpress.org/news/2018/12/bebo/), which was released in December 2018.
```

File: docs/designers-developers/faq.md [hunk 2]
```diff
@@ -243,9 +243,9 @@ Here is a brief animation illustrating how to find and use the keyboard shortcut
 
 No. [TinyMCE](https://www.tinymce.com/) is only used for the "Classic" block.
 
-## What browsers will Gutenberg support?
+## What browsers do Gutenberg support?
```
Comment (chrisvanpatten) [inline docs/designers-developers/faq.md]:
```suggestion
## What browsers does Gutenberg support?
```

File: docs/designers-developers/faq.md [hunk 3]
```diff
@@ -267,39 +267,39 @@ We plan to continue supporting existing meta boxes while providing new ways to e
...
 
-## Will there be columns?
+## Does Gutenberg support columns?
 
 Yes, a columns block is available in Gutenberg.
 
-## Will there be nested blocks?
+## Does Gutenberg support nested blocks?
 
 We are currently implementing the infrastructure for nested blocks support. We expect this to open further customization opportunities. Block authors also can nest components and HTML inside of a block during construction. The UI for manipulating nested blocks is still being refined, and depending on the timing, it might not be included in the first version of Gutenberg.
```
Comment (chrisvanpatten) [inline docs/designers-developers/faq.md]: This answer will probably need a rewrite; nested blocks are live and in Gutenberg.

**Review by chrisvanpatten — DISMISSED** [review]
Great update! Had a few small grammar nits.

File: docs/designers-developers/faq.md (hunk 3 above)
Comment (maximejobin) [reply docs/designers-developers/faq.md]: What about:

```suggestion
Yes, it is supported. You can have multiple levels of nesting – blocks within blocks within blocks.
```

File: docs/designers-developers/faq.md [hunk 4]
```diff
@@ -267,39 +267,39 @@ We plan to continue supporting existing meta boxes while providing new ways to e
...
 
 Indeed. There are multiple ways in which custom post types can leverage Gutenberg. The plan is to allow them to specify the blocks they support, as well as defining a default block for the post type. It's not currently the case, but if a post type disables the content field, the “advanced” section at the bottom would fill the page.
 
-## Will there be columns?
+## Does Gutenberg support columns?
 
 Yes, a columns block is available in Gutenberg.
 
-## Will there be nested blocks?
+## Does Gutenberg support nested blocks?
```
Comment (gziolo) [inline docs/designers-developers/faq.md]: The answer also needs to get updated as nested blocks are part of Gutenberg.

**Review by gziolo — APPROVED** [review]

File: docs/designers-developers/faq.md (hunk 4 above)
Comment (maximejobin) [reply docs/designers-developers/faq.md]: I don't understand why my suggestion/answer is above the question.

Comment (maximejobin) [reply docs/designers-developers/faq.md]: I already suggested this answer in a previous comment but it seems it didn't go where it was supposed to. Let me put it here again:

```suggestion
Yes, it is supported. You can have multiple levels of nesting – blocks within blocks within blocks.
```

File: docs/designers-developers/faq.md (hunk 3 above)
Comment (gziolo) [reply docs/designers-developers/faq.md]: Looks good. Can you include it in this PR?

File: docs/designers-developers/faq.md (hunk 4 above)
Comment (gziolo) [reply docs/designers-developers/faq.md]: I see it now. Can you include it in the patch? I will merge it afterward.

**Review by Soean — APPROVED** [review]
All requested changes were added to this PR. So we can merge it.

**Comment by gziolo** [comment]
@maximejobin, congrats on your first contribution. This change should be applied a few months back 😅

**Comment by maximejobin** [comment]
@gziolo Thank you for your words!

**Pull request merged**